suricata/update/parsers.py

```diff
@@ -56,13 +56,7 @@
       'help': "Override Suricata version"}),
     (("--user-agent",),
      {'metavar': '<user-agent>',
-      'help': "Set custom user-agent string"}),
-    (("--no-check-certificate",),
-     {'action': 'store_true', 'default': None,
-      'help': "Disable server SSL/TLS certificate verification"}),
```

Comment:
I don't see this re-added anywhere.  It either needs to remain global, or added to any command that may go out on the network.